scripts/cleanOldStats.ts

```
@@ -1,11 +1,13 @@
 import 'dotenv/config';
 import { getDb } from '../lib/db';
 import { PoolStats } from '../lib/entities/PoolStats';
+import { User } from '../lib/entities/User';
 import { UserStats } from '../lib/entities/UserStats';
+import { Worker } from '../lib/entities/Worker';
 import { WorkerStats } from '../lib/entities/WorkerStats';
-import { LessThan } from 'typeorm';
+import { In, Not, LessThan } from 'typeorm';
```

ESLint

[error] `typeorm` import should occur before import of `../lib/db`

(import/order)